warehouse/admin/templates/admin/organization_applications/detail.html

```diff
@@ -388,10 +388,31 @@ <h3 class="card-title">Organization Request{% if information_requests %}{% if ou
               </label>
               <div class="col-12">
                 <table class="table">
-                  <tr><th>Application</th><th>Submitted</th><th>Requestor</th></tr>
+                  <tr><th>Application</th><th>Status</th><th>Submitted</th><th>Requestor</th></tr>
                   {% for application in conflicting_applications %}
                     <tr>
                       <td><a target="_blank" href="{{ request.route_url('admin.organization_application.detail', organization_application_id=application.id) }}">{{ application.name }}</a></td>
+                      <td>{{ application.status }}</td>
+                      <td>{{ application.submitted|format_date() }}</td>
+                      <td><a target="_blank" href="{{ request.route_url('admin.user.detail', username=application.submitted_by.username) }}">{{ application.submitted_by.username }}</a></td>
+                    </tr>
+                  {% endfor %}
+                </table>
+              </div>
+            </div>
+            {% endif %}
+            {% if conflicting_namespace_prefixes %}
+            <div class="form-group">
+              <label class="col-12 control-label">
+                Conflicting Prefixes <i class="fa fa-clipboard-question text-orange"></i>
+              </label>
+              <div class="col-12">
+                <table class="table">
+                  <tr><th>Application</th><th>Status</th><th>Submitted</th><th>Requestor</th></tr>
+                  {% for application in conflicting_namespace_prefixes %}
+                    <tr>
+                      <td><a target="_blank" href="{{ request.route_url('admin.organization_application.detail', organization_application_id=application.id) }}">{{ application.name }}</a></td>
+                      <td>{{ application.status }}</td>
```

Comment:
I think we could probably just replace the 'conflicting applications' section with this rather than potentially duplicating it here, it should be clear enough which are exact matches & which are prefixes?

Reply:
smart. 71d5f4b2777327b1b6179200c7953b0f4207b61b